OJP/OJP_Trips.xsd

```diff
@@ -722,6 +727,25 @@
 			<xs:enumeration value="upAndDown"/>
 			<xs:enumeration value="downAndUp"/>
 		</xs:restriction>
+	</xs:simpleType>
+		<xs:simpleType name="AccessibilityFeaturesTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for AccessibilityFeatures.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="visualSigns"/>
+			<xs:enumeration value="tactileGuidingStrips"/>
+			<xs:enumeration value="tactileOrAuditorySigns"/>
+			<xs:enumeration value="elevatorAccousticAnnouncements"/>
+			<xs:enumeration value="elevatorTactileButtons"/>
+			<xs:enumeration value="elevatorSuitableForCycles"/>
+			<xs:enumeration value="entranceLevelAccess"/>
+			<xs:enumeration value="entranceLiftInVehicle"/>
```

Comment:
not sure what you exactly mean with **entranceLiftInVehicle**

Reply:
Would a convention like the following be better:
elevator**Has**TactileButtons
elevator**Is**SuitableForCycles
vehicleEntrance**Has**LevelAccess
vehicleEntrance**Has**LiftInVehicle ?

Reply:
@trurlurl @Aurige  can you come up with a synchronised list for us to incorporate?

Reply:
See my proposal with all enums below - the proposal below fits well with the other, additional enums that we might want to add.

Reply:
was integrated by André